classes/05class/exercises/c05-3m01/nimmi89/Makefile

```diff
@@ -0,0 +1,26 @@
+# Makefile
+BUCKET_NAME ?= test
+COMPOSE_RUN_EXERCISE = docker-compose run --rm c05-3m01
+
+create_bucket: 
+	BUCKET_NAME=$(BUCKET_NAME) $(COMPOSE_RUN_EXERCISE) make _test_create_bucket
```

Comment:
You may have many variables here and it becomes unreadable in Makefile.

Ideally, you'd pass the variables using the .env file and setting it in docker-compose.

For that you needed to export the variable: `export BUCKET_NAME ?= test`